mlir/test/Dialect/LLVMIR/invalid.mlir

```diff
@@ -439,6 +439,105 @@ llvm.func @scalable_vec_requires_splat() -> vector<[4]xf64> {
   llvm.return %0 : vector<[4]xf64>
 }
 
+
+// -----
+
+llvm.func @int_attr_requires_int_type() -> f32 {
+  // expected-error @+1 {{expected integer type}}
```

Comment:
```suggestion
  // expected-error @below {{expected integer type}}
```
ulta nit: can you use below everywhere in the moved tests? It is preferred over @+1